src/frame/mod.rs

```diff
@@ -46,6 +47,10 @@ impl Frame {
                 // some link frames are allowed to have the same id as long their content is different
                 if id == "WCOM" || id == "WOAR" {
                     self.content.link() == other.content.link()
+                }
+                // Multiple UFID frames are allowed to exist as long as they have different owner_identifiers
+                else if id == "UFID" {
+                    self.content.ufid() == other.content.ufid()
```

Comment:
This is handled by `self.content.unique() == other.content.unique()`, so please remove this block